src/components/annotator/TextDisplay.vue

```diff
@@ -42,6 +42,11 @@ export default class TextDisplay extends Vue {
 
 	public created() {
 		this.currentAppellations = this.appellations;
+		// store.subscribe((mutation: any, state: any) => {
```

Comment:
If this isnt needed remove it. Theres no point leaving commented code hanging around

Reply:
Removed